doc/project/committers.md

```diff
@@ -35,3 +22,28 @@ This may happen in cases such as a committer failing to act as a good citizen wi
 It may also be a natural function of "pruning the tree" as an individual's involvement in the project changes over time.
 
 The list of committers is maintained within the project's Git repository and is available in the [COMMITTERS file](https://github.com/lowRISC/opentitan/blob/master/COMMITTERS) in the repository root.
+
+## Commit Escalation Guidelines
+
+When reviewing a pull request, there are a range of options in giving feedback.
+Although committers have the final say on patch approval or requiring changes, the task of reviewing is shared by all contributors, who may make similar requests.
+Options for reviewing a pull request include:
+* Approval, with no further conditions.
+  * This is appropriate when you are confident the changes are correct, sufficiently explained, in line with the project coding styles, and don't need further review by others or a companion [RFC]({{< relref "rfc_process" >}}) to be written.
+    You should only use this for cases where you have sufficient expertise in the areas being modified.
+  * Note that if the PR came from someone without commit rights, you will need to rebase and merge for them.
+* Approval, but with a request to get an additional approval from other named reviewers.
+  * This is appropriate when you believe the changes are correct, but would either like a second opinion or to ensure that another contributor is aware of and approves of the changes.
```

Comment:
```suggestion
  * This is appropriate when you believe the changes are correct, but would either like a second opinion or ensure that another contributor is aware of and approves of the changes.
```

Reply:
I think the original is more accurate, but I'm no grammarian. The former would expand as "but would like a second opinion or would like to ensure that ..."